binder-gallery.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: CMIP6 Ocean Biogeochemistry
+description: >-
+  This project examines historical simulations and future projections
+  of ocean carbon uptake and storage in the CMIP6 ensemble,
+  and explore their impacts on marine ecosystems drivers,
+  with a focus on primary productivity.
```

Comment:
@mara-freilich, let me know if this description of the repo is accurate. I copied it from the README. Also, Pangeo Gallery is going to use this [image](https://github.com/andersy005/cmip6hack-ocean-bgc/blob/master/thumbnail.png) as a thumbnail. Let me know if there is a better image we can use as a thumbnail.

Reply:
Here's a slightly edited version: "This project examines historical simulations and future projections of ocean carbon uptake and storage in the CMIP6 ensemble, and explores the impact of carbon dioxide emissions on marine ecosystems drivers, with a focus on primary productivity."

Thanks! I think that's a good image to use as a thumbnail